src/main/io/rcdevice.c

```diff
@@ -270,6 +270,10 @@ static void runcamDeviceParseV2DeviceInfo(rcdeviceResponseParsingContext_t *ctx)
 // definition of runcamDeviceInfo_t to know more)
 static void runcamDeviceGetDeviceInfo(runcamDevice_t *device)
 {
+    // preset with rc split 2s support which has issue to respond
```

Comment:
This probably should not be blindly enabled, as we are asking the device for information that should populate the features.

We should probably fix the protocol differences with this camera, so we can read the featrures correctly, or at least have a configuration variable to enable this hack.

Reply:
Change it as you like it is hot fix for now. Having feature variable would be good for diagnostics via CLI.Also I noticed that runcam attempt interval variable was removed in some Inav version as there was related workaround before.
I would add possibility to set features manually and let user to configure/map what he needs.